WalletWasabi.Tests/UnitTests/Transactions/TransactionStoreTests.cs

```diff
@@ -29,9 +29,7 @@ public async Task CanInitializeAsync()
 		[Fact]
 		public async Task CanDoOperationsAsync()
 		{
-			await using var txStore = new TransactionStoreMock();
-			var network = Network.Main;
-			await txStore.InitializeAsync(network);
+			var txStore = await CreateTransactionStoreAsync();
```

Comment:
```suggestion
			await using var txStore = await CreateTransactionStoreAsync();
```

Reply:
I don't get it. `await CreateTransactionStoreAsync()` returns a `TransactionStore` which is not awaitable and does not implement `IDisposable`.

Reply:
That is true but it is implementing IAsyncDisposable.